Gemfile

```diff
@@ -83,6 +83,8 @@ gem "rails-i18n"
 
 # Provides helper methods to easily add 'active' tag on links
 gem "active_link_to"
+
+# Provides named content areas to Action View through `partial` method
```

Comment:
I was v confused with the _pages partial component logic, since I did not see ViewComponents in our gem file. I am hoping this comment will help others in a similar situation. I think it may be good enough to help others, but we may consider noting this gem somewhere else too.